### ISSUE TITLE
Make it clear that the  `package.json` and `package-lock.json` Should not be committed

### ISSUE THREAD

**Issue by NS007-dev** [EddieHubCommunity/EventCalendar#26]
I have seen in some pull requests that the package.json is being committed as well, maybe it should be in the .gitignore files or should made clear to not add the package files.
Thank you.

**Comment by eddiejaoude** [comment]
We should make this clear not to commit, these should not be ignored otherwise people cannot install in the dependencies

**Comment by mohan-murali** [comment]
yes otherwise their might be different version of packages installed for different people.

**Comment by NS007-dev** [comment]
> We should make this clear not to commit, these should not be ignored otherwise people cannot install in the dependencies

Yes I agree

**Comment by NS007-dev** [comment]
I will change the issue name. 
Thank you

**Comment by eddiejaoude** [comment]
thank you 👍

**Comment by suitupstan** [comment]
Is this to be added in the readme?

**Comment by NS007-dev** [comment]
Yes, I think that is where it should be said @krishguptadev

**Comment by suitupstan** [comment]
I think we should wait for the contributing.md file and add it there instead. Thanks 😊👍
